client-app/shared/catalog/components/product-card-list.vue

```diff
@@ -78,9 +78,9 @@
           <svg
             class="size-3 shrink-0"
             :class="{
-              'text-[color:var(--color-success)]': true,
-              'text-[color:var(--color-warning)]': false,
-              'text-[color:var(--color-danger)]': false,
+              'text-success': true,
```

Comment:
text-success could be moved to class and other could be removed

Reply:
done